docs/build-docs.sh

```diff
@@ -43,7 +43,7 @@ rm -rf docs/build
 mkdir docs/build/
 cp -r docs/_templates docs/conf.py docs/*.svg docs/*.json  docs/build/
 
-sphinx-build -c docs/ -WETan -j auto -D language=en docs/build/ docs/build/html
+sphinx-build -c docs/ -ETan -j auto -D language=en docs/build/ docs/build/html
```

Comment:
Hey! It passes sphinx execution through in case of warnings. In the cast of the flag `-W`, a sphinx process is being terminated on warning. 
The proposal to remove the flag was motivated by the issue it causes when running sphinx to generate docu for [avro_dataset.py](https://github.com/kislerdm/kedro/blob/feature/issue-607_adds_avro_dataset/kedro/extras/datasets/pandas/avro_dataset.py).

Reply:
Ah I see, thanks for explaining. We do actually want to keep the `-W` flag, because for our building of the docs the process should terminate on warning and not continue.